brooklyn-server/karaf/features/src/main/history/dependencies.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
is this file now removed from source control?  i know there was talk of this, but here it looks like it is being added back in.